### DISCUSSION TITLE
Does Legendary have the equivalent files to the .item manifests?

### DISCUSSION THREAD

**Discussion by macaquinyho** [legendary-gl/legendary#276] [Q&A]
Hi, I was wondering if Legendary stored the manifests in the same way as the Epic Launcher because a Rocket League mod (bakkesmod) gets the version info from `C:\ProgramData\Epic\EpicGamesLauncher\Data\Manifests` and obviously Legendary doesn't use that path, so the version check fails and the mod doesn't launch. If Legendary has this, it would be trivial to just add the check to the equivalent folder.

I looked and it seems that the manifests are stored in `C:\Users\*users*\.config\legendary\manifests` but they are .manifest files instead of the .item files that are in `C:\ProgramData\Epic\EpicGamesLauncher\Data\Manifests`.

**Comment by derrod** [comment]
It doesn't. All metadata for installed games is stored inside `installed.json` and the respective metadata folders instead. You can however enable EGL sync, which will create the `.item` files in that location. It doesn't actually check whether EGS is isntalled, the `C:\ProgramData\Epic\EpicGamesLauncher\Data\Manifests` folder just needs to exist.

**Reply by macaquinyho** [reply]
That worked! 
Thanks for the answer.

**Reply by cbartondock** [reply]
Will eventually make a separate Legendary Parser for Steam ROM Manager, but for now I am just suggesting users do this.
